src/redux/initialState.js

```diff
@@ -6,6 +6,16 @@ const initialState = {
     { id: 'table', name: 'Table' },
     { id: 'dining', name: 'Dining' },
   ],
+  galleryTabs: [
+    { id: 'featured', name: 'Featured' },
+    { id: 'topSeller', name: 'Top seller' },
+    { id: 'saleOff', name: 'Sale off' },
+    { id: 'topRated', name: 'Top rated' },
+  ],
+  galleryPromotedProduct: {
+    src:'https://www.pngfind.com/pngs/m/55-553255_double-bed-png-bed-png-transparent-png.png',
```

Comment:
Poszukaj proszę innego zdjęcia, ablo nawet niech będze narazie z pełnym tłem bo to akurat zdjęcie nie  prezentuje się za dobrze, teoretycznie nazwa sugeruje przeźroczyste tło, ale tak nie jest

![image](https://user-images.githubusercontent.com/7238770/96395165-2f508380-11c4-11eb-8d18-bb81d2ea9f43.png)

Reply:
Łukasz, link poprawiony.